src/index.html

```diff
@@ -16,7 +16,7 @@
     <link href='https://accounts.google.com' rel='preconnect'>
     <link href='https://ssl.gstatic.com' rel='preconnect'>
 
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700' rel='stylesheet' type='text/css'>
```

Comment:
we're only using 200, 400/normal and 700/bold, and no italic variants in our styles (aside the settings page, but that's only for bold.) if we pair this back down we'll see slightly faster load time for this typeface & its font-weights.

we should also see where we're using 200 and what it looks like compared to 300, as that is what we're pulling down.